src/contracts/pods/EigenPod.sol

```diff
@@ -91,9 +91,12 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     /// @notice This variable tracks any ETH deposited into this contract via the `receive` fallback function
     uint256 public nonBeaconChainETHBalanceWei;
 
-     /// @notice This variable tracks the total amount of partial withdrawals claimed via merkle proofs prior to a switch to ZK proofs for claiming partial withdrawals
+    /// @notice This variable tracks the total amount of partial withdrawals claimed via merkle proofs prior to a switch to ZK proofs for claiming partial withdrawals
     uint64 public sumOfPartialWithdrawalsClaimedGwei;
 
+    /// @notice Number of validators with proven withdrawal credentials, who do not have proven full withdrawals
+    uint256 activeValidatorCount;
```

Comment:
we should make visibility explicit
I would think 'public' for sake of simplicity

Reply:
public adds bytecode

i'm fine with this but... i figured we should stop defaulting to public

idk maybe this is important?

Reply:
actually ive decided

public is worse because if people use it we can't ever get rid of it

let's make it public IFF we release the partial withdrawal feature that uses this var.